app/scripts/modules/core/src/pipeline/config/triggers/pipeline/pipelineTriggerOptions.directive.spec.js

```diff
@@ -41,6 +43,7 @@ describe('Pipeline Trigger: PipelineTriggerOptionsCtrl', function() {
   it('loads executions on initialization, setting state flags', function () {
     let executions = [];
     spyOn(executionService, 'getExecutionsForConfigIds').and.returnValue($q.when(executions));
+    spyOn(executionsTransformer, "addBuildInfo");
```

Comment:
single quotes! (same for the other lines below)

Reply:
Fixed